core/templates/pages/partnerships-page/partnerships-page.component.ts

```diff
@@ -67,6 +68,19 @@ export class PartnershipsPageComponent implements OnInit, OnDestroy {
     this.pageTitleService.setDocumentTitle(translatedTitle);
   }
 
+  getFormLink(): string {
+    let userLang = this.translateService.currentLang;
+
+    if (userLang === 'en' || userLang === 'pcm' || userLang === 'kab') {
```

Comment:
Don't do this -- it doesn't handle cases where new languages are added in the future. The logical flow should be: "if en, do this, else if pt-br, do this, else do this".

The else clause can presumably cover things like pcm and kab too, right? (In that the forms link will be shown but it will show up in English.)

Reply:
Ah ok, thanks. OK, what you have seems fine then (logically).